CRM/UF/Form/Field.php

```diff
@@ -536,51 +536,51 @@ public function postProcess() {
       $name = $this->_selectFields[$params['field_name'][1]];
     }
 
-    //Hack for Formatting Field Name
+    // Hack for Formatting Field Name.
     if ($params['field_name'][0] == 'Formatting') {
       $params['field_name'][1] = 'formatting_' . rand(1000, 9999);
     }
 
-    //check for duplicate fields
-    if ($params["field_name"][0] != "Formatting" && CRM_Core_BAO_UFField::duplicateField($params, array('uf_group' => $params['uf_group_id'], 'uf_field' => $params['id']))) {
-      CRM_Core_Session::setStatus(ts('The selected field already exists in this profile.'), ts('Field Not Added'), 'error');
-      return;
+    try {
```

Comment:
@eileenmcnaughton Please have a look at whether this looks fine whenever you get the time.